tests/unit/type/incr.tcl

```diff
@@ -9,6 +9,10 @@ start_server {tags {"incr"}} {
         r incr novar
     } {2}
 
+    test {DECR against key created by incr} {
+        r decr novar
+    } {1}
```

Comment:
i'm curious, why was this added?

Reply:
there isn't a single (deterministic) test that executes DECR